hazelcast/src/main/java/com/hazelcast/map/impl/EntryViews.java

```diff
@@ -46,49 +46,45 @@ private EntryViews() {
     }
 
     public static <K, V> EntryView<K, V> createSimpleEntryView(K key, V value, Record record) {
-        final SimpleEntryView<K, V> simpleEntryView = new SimpleEntryView<K, V>(key, value);
-        simpleEntryView.setCost(record.getCost());
-        simpleEntryView.setVersion(record.getVersion());
-        simpleEntryView.setHits(record.getHits());
-        simpleEntryView.setLastAccessTime(record.getLastAccessTime());
-        simpleEntryView.setLastUpdateTime(record.getLastUpdateTime());
-        simpleEntryView.setTtl(record.getTtl());
-        simpleEntryView.setCreationTime(record.getCreationTime());
-        simpleEntryView.setExpirationTime(record.getExpirationTime());
-        simpleEntryView.setLastStoredTime(record.getLastStoredTime());
-        return simpleEntryView;
+        return new SimpleEntryView<K, V>(key, value)
+                .setCost(record.getCost())
+                .setVersion(record.getVersion())
+                .setHits(record.getHits())
+                .setLastAccessTime(record.getLastAccessTime())
+                .setLastUpdateTime(record.getLastUpdateTime())
+                .setTtl(record.getTtl())
+                .setCreationTime(record.getCreationTime())
+                .setExpirationTime(record.getExpirationTime())
+                .setLastStoredTime(record.getLastStoredTime());
     }
 
     public static <K, V> EntryView<K, V> createLazyEntryView(K key, V value, Record record,
                                                              SerializationService serializationService,
                                                              MapMergePolicy mergePolicy) {
-        LazyEntryView<K, V> lazyEntryView = new LazyEntryView<K, V>(key, value, serializationService, mergePolicy);
-        lazyEntryView.setCost(record.getCost());
-        lazyEntryView.setVersion(record.getVersion());
-        lazyEntryView.setHits(record.getHits());
-        lazyEntryView.setLastAccessTime(record.getLastAccessTime());
-        lazyEntryView.setLastUpdateTime(record.getLastUpdateTime());
-        lazyEntryView.setTtl(record.getTtl());
-        lazyEntryView.setCreationTime(record.getCreationTime());
-        lazyEntryView.setExpirationTime(lazyEntryView.getExpirationTime());
```

Comment:
The expiration and last stored time were taken from the `lazyEntryView` itself, not from the `record`.